docs/topics/mta-web-config-jira-connection.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-config-jira-connection_{context}"]
+= Configuring Jira connection
+
+To be able to create Jira issues from within the {ProductName} ({ProductShortName}) {WebName} and to track application migrations, you need to create a Jira instance in {ProductShortName} and establish a connection to that instance.
+
+.Procedure
+
+. In *Administration* view, under *Issue Management*, click *Jira*.
++
+The *Jira configuration* page opens.
+. Click *Create new*.
++
+The *New instance* window opens.
+. Enter the following information:
+* Name of the instance
+* URL of the web interface of your Jira account
+* Instance type - select either *Jira Cloud* or *Jira Server/Datacenter* from the drop-down list
+* Credentials - select from the drop-down list
++
+[NOTE]
+====
+If the selected instance type is *Jira Cloud*, only *Basic Auth* credentials are displayed in the list.
+
+If the selected instance type is *Jira Server/Datacenter*, both *Basic Auth* and *Token Bearer* credentials are displayed. Which type you need to choose depends on the configuration of the particular Jira server or datacenter.
+====
++
+. By default, a connection cannot be established with a server with an invalid certificate. To override this restriction, toggle the *Enable insecure communication* switch.
+. Click *Create*.
++
+The new connection instance appears on the *Jira configuration* page.
++
+Once the connection has been established and authorized, the status in the *Connection* column becomes *Connected*.
++
+If the *Connection* status becomes *Not connected*, click the status to see the reason for the error.
+
```

Comment:
Suggestion "If the connection has the status `Not connected`,..." [or *Not connected*]